.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
please remove this file

Reply:
Apologies for missing that - file is removed.